serverless.yml

```diff
@@ -7,9 +7,24 @@ provider:
   name: aws
   runtime: nodejs20.x
   region: eu-north-1
+  stage: ${opt:stage, 'production'}
```

Comment:
Add env variable as the first option, then you can easily give it as a command prefix when deploying locally